torch/csrc/autograd/functions/basic_ops.cpp

```diff
@@ -5,6 +5,8 @@
 #include "torch/csrc/autograd/functions/utils.h"
 #include "torch/csrc/utils/auto_gpu.h"
 
+#include <ATen/Tensor.h>
```

Comment:
Only `#include <ATen/ATen.h>` is actually correct, for not-so-good but not-easy-to-fix reasons. We may eventually move all the other headers so  ATen/ATen.h is the only thing that works.